src/main/java/ua/team3/carsharingservice/model/Payment.java

```diff
@@ -0,0 +1,51 @@
+package ua.team3.carsharingservice.model;
+
+import jakarta.persistence.Basic;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "payments")
+public class Payment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(EnumType.STRING)
+    @Basic(optional = false)
+    private Status status;
+    @Enumerated(EnumType.STRING)
+    @Basic(optional = false)
+    private Type type;
+    @OneToOne(fetch = FetchType.EAGER)
+    private Rental rental;
+    @Column(nullable = false)
+    private String sessionUrl;
+    @Column(nullable = false, unique = true)
+    private String sessionId;
+    @Column(nullable = false)
+    private BigDecimal amount;
+
+    public enum Status {
+        PENDING,
+        PAID
+    }
+
+    public enum Type {
+        PAYMENT,
+        FINE,
+        EXPIRED
```

Comment:
EXPIRED should be in Status enum, as I understand

Reply:
Yes, you right